schema-extended/arxiv-extended.json

```diff
@@ -0,0 +1,707 @@
+                 {"$ref": "#/definitions/language"},
+                 {"not": {"enum": ["en"]}}
+             ]
+            }
+          },
+          "additionalProperties": false
+        }
+      },
+      "related_link": {
+        "tile": "Related link",
+        "description": "Author-provided link to related resource.",
+        "type": "object",
+        "properties": {
+          "uri": {
+            "type": "string",
+            "format": "uri"
+          },
+          "display_text": {
+            "description": "Descriptive text to associate with the URI.",
+            "type": "string",
+            "minLength": 1
+          },
+          "category": {
+            "title": "Link category",
+            "description": "The type of resource the link resolves to.",
+            "type": "string",
+            "$comment": "TODO: refine this enumeration",
+            "enum": ["related article", "slides", "notes", "software", "dataset", "simulation", "video", "review", "news item", "personal website", "blog entry", "other"]
+          }
+        },
+        "required": [
+          "uri",
+          "display_text",
+          "category"
+        ]
+      },
+      "unversioned_identifier": {
+        "title": "arXiv canonical identifier",
+        "type": "string",
+        "pattern": "^(([0-9]{4}\\.[0-9]{4,5})|([a-z\\-]+\\/[0-9]{2}[01][0-9]{4}))$"
+      },
+      "versioned_identifier": {
+        "title": "arXiv canonical identifier with version affix",
+        "type": "string",
+        "pattern": "^(([0-9]{4}\\.[0-9]{4,5})|([a-z\\-]+\\/[0-9]{2}[01][0-9]{4}))v[0-9]+$"
+      },
+      "version": {
+
+        "type": "object",
+        "properties": {
+          "identifier": {
+            "$ref": "#/definitions/versioned_identifier"
+          },
+          "document_types": {
+            "title": "List of document types",
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/document_type"
+            },
+            "minItems": 1,
+            "uniqueItems": true
+          },
+          "submitter_info": {
+            "title": "Submitter",
+            "description": "Information about the submitter and possible proxy for the submitter.",
+            "type": "object",
+            "properties": {
+              "submitter": {
+                "$ref": "#/definitions/person"
+              },
+              "proxy_name": {
+                "title": "Proxy name",
+                "description": "Name of the proxy that submitted on the submitter's behalf.",
+                "type": "string",
+                "minLength": 1
+              }
+            },
+            "required": [
+              "submitter"
+            ]
+          },
+          "submitted_date": {
+            "title": "Submitted date",
+            "description": "Date this version was submitted.",
+            "type": "string",
+            "format": "date-time"
+          },
+          "announced_date": {
```

Comment:
is this present also if the eprint is not announced, because there are too many versions?